third_party/Makefile

```diff
@@ -239,3 +239,10 @@ ifneq (,$(wildcard msccl/tests/msccl-tests-nccl/Makefile))
 	mkdir -p $(SB_MICRO_PATH)/bin/msccl-tests-nccl && \
 	cp -r -v ./msccl/tests/msccl-tests-nccl/build/* $(SB_MICRO_PATH)/bin/msccl-tests-nccl/
 endif
+
+# Build nvbandwidth.
+nvbandwidth: sb_micro_path
+ifneq (,$(wildcard nvbandwidth/debian_install.sh))
+	cd ./nvbandwidth && ./debian_install.sh && cd ..
```

Comment:
maybe run `cmake . && make` directly? dependencies have already been installed in dockerfile and may be broken when re-install